google/resource_endpoints_service.go

```diff
@@ -0,0 +1,297 @@
+package google
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/servicemanagement/v1"
+)
+
+func resourceEndpointsService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceEndpointsServiceCreate,
+		Read:   resourceEndpointsServiceRead,
+		Delete: resourceEndpointsServiceDelete,
+		Update: resourceEndpointsServiceUpdate,
+		Schema: map[string]*schema.Schema{
+			"service_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"openapi_config": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"grpc_config": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"protoc_output": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"config_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+				ForceNew: true,
```

Comment:
Which is this ForceNew if it is only an exported field?

Reply:
You're right, that makes no sense.  Removed - you can't get a new config_id unless the configs change, anyway.